src/index.js

```diff
@@ -13,6 +13,17 @@ export default class TC_Wrapper {
     return this.instance;
   }
 
+
+  /**
+   * Function isString from Lodash : https://github.com/lodash/lodash/blob/4.17.15/lodash.js#L12135
```

Comment:
In the end, it's not the one from lodash which is far more complete, so better not specify this part